src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+
+  if (days < 3) {
+    return dailyRent * days;
+  } else if (days < 7) {
```

Comment:
you don't need else after if with return

Reply:
Done!